Add comment

ad4565 opened the pull request.

ad4565 committed:
Update README.md

ryan-bello commented:
I think you're supposed to leave a comment regarding the article according to the instructions. Maybe just some part of it you found interesting, or if you agree with my thoughts. Also, could you rename the commit to something like "added my comment", and create a separate section in the markdown for your comment?

ad4565 committed:
Update README.md

I was also surprised to learn developer's don't necessarily have full ownership of their side projects

ryan-bello commented:
Could you please also add a heading before your comment, just so it's clearly distinguished from what I wrote? The way it looks right now kinda seems like I'm quoting you haha. Just add the following line before your comment
`## Comment`

maya-b-coding committed:
Added a comment heading
